frontend/src/Metamaps/Util.js

```diff
@@ -135,9 +135,26 @@ const Util = {
   },
   mdToHTML: text => {
     const safeText = text || ''
+    const parsed = new Parser().parse(safeText)
+
+    // remove images to avoid http content in https context
+    const walker = parsed.walker()
+    for (let event = walker.next(); event = walker.next(); event) {
+      const node = event.node
+      if (node.type === 'image') {
+        const imageAlt = node.firstChild.literal
+        const imageSrc = node.destination
+        const textNode = new Node('text', node.sourcepos)
+        textNode.literal = `![${imageAlt}](${imageSrc})`
+
+        node.insertBefore(textNode)
+        node.unlink() // remove the image, replacing it with markdown
+        walker.resumeAt(textNode, false)
+      }
+    }
+
```

Comment:
this is not necessarily related to this PR. I probably shouldn't have put these together :(